devito/builtins.py

```diff
@@ -343,7 +343,7 @@ def buff(i, j):
         options = None
 
     # TODO: Figure out why yask doesn't like it with dse/dle
-    assign(lhs, rhs, options=options, name=name, dse='noop', dle='noop')
+    assign(lhs, rhs, options=options, name=name, dse='advanced', dle='advanced')
```

Comment:
no, you should just drop them, because the defaults are advanced already

also, the comment "TODO: ..." above should be dropped